File: 07. Perulangan Lanjut/C. Pola III/c_pola_iii.cpp
#include <iostream>
int main(){
  int N, counter = 0;
  std::cin >> N;
  for (int i = 0; i < N; i++){
    for (int j = 0; j < i + 1; j++){
      std::cout << counter % 10;
      counter++;
    }
    std::cout << std::endl;
  }
  return 0;
}

File: 07. Perulangan Lanjut/C. Pola III/c_pola_iii.pas
program c_pola_iii;
var
  N, i, j, counter : integer;
begin
  counter := 0;
  readln(N);
  for i := 1 to N do begin
    for j := 1 to i do begin
      write(counter mod 10);
      counter := counter + 1;
    end;
    writeln();
  end;
end.
